### ISSUE TITLE
coolprop.org down again

### ISSUE THREAD

**Issue by ibell** [CoolProp/CoolProp#1882]
@Rod-Persky can you investigate?  I think we expired yesterday?

**Comment by Rod-Persky** [comment]
Hi @ibell, I don't seem to have the login credentials any longer. I wonder if I sent them to you any time?

**Comment by ibell** [comment]
Yeah, I tried to reset the password, thinking that maybe I was added as a
user, but that didn't work.

**Comment by Rod-Persky** [comment]
Ok, looking into the password reset process - although hopefully we didn't set the account email to a coolprop domain

**Comment by ibell** [comment]
I forwarded you the email that you sent me before with the login info.  If
you don't get it, please get in contact with me at ian.h.bell@gmail.com

**Comment by Rod-Persky** [comment]
Sorted @ibell should be updated soon

**Comment by ibell** [comment]
*phew*...  To avoid future drama, any chance you could add a few of us as
admins?

**Comment by Rod-Persky** [comment]
Beyond sharing the credentials to the name.com I am unaware of how to handle password distribution.

Happy to see password distribution or additional accounts get some traction 👍

**Comment by ibell** [comment]
@Rod-Persky Yeah, I would be interested in at the very least sharing credentials with myself, @jowr and @thorade , and if there is a backup contact that can be attached to the account, that would be great too.

**Comment by ibell** [comment]
Closing as part of an issue-tracker cleanup — this issue has had no activity in over two years.

*To be transparent: this comment and the close action are being performed by Claude (Anthropic's AI coding assistant), running on behalf of @ibell. Ian reviewed and approved the bulk-close criteria; I am the one executing the action.*

If you would like this issue reconsidered, please feel free to **re-open it** — a fresh comment with current behaviour against the latest CoolProp release is the most useful way to revive the discussion.

— Claude (acting for @ibell)
